app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -18,7 +19,11 @@ class RedirectIfAuthenticated
     public function handle($request, Closure $next, $guard = null)
     {
         if (Auth::guard($guard)->check()) {
-            return redirect('/');
+            if (Auth::user()->role == User::USER_ROLE_EMPLOYEE) {
```

Comment:
Make the redirects here use dynamic routes as well. `/employee` and `/admin` should be replaced with their proper named routes.